types/signed_header.go

```diff
@@ -137,9 +127,19 @@ func (sh *SignedHeader) ValidateBasic() error {
 		return err
 	}
 
-	// Rollkit vA uses a centralized sequencer.
-	if err := sh.checkCentralizedSequencer(); err != nil {
-		return err
+	// Rollkit vA uses a centralized sequencer, so there should only be one validator
```

Comment:
I'm fine with this, but I was previously asked to separate this into its own function

Reply:
Yes, I understand and have that context too. I think it's fine to roll with this.